### ISSUE TITLE
Genre background images do not display

### ISSUE THREAD

**Issue by pikap3w** [nukeop/nuclear#322]
Genre images do not display on version 0.4.5 or version 0.4.4. This also hides the text so you cannot see which genre the square represents.

![Screen Shot 2019-04-29 at 1 48 22 PM](https://user-images.githubusercontent.com/29871623/56919219-ac92b080-6a85-11e9-9f7a-beaa0bb758bf.png)

I traced this to an issue with the request to the picsum image generator API. See API reference guide here: https://picsum.photos/.

**Example and Suggested Fix:**

_Current_
```
Request URL: https://picsum.photos/256x256/?random&seed=0
Request Method: GET
Status Code: 404 
```
_Suggested Fix_
```
Request URL: https://picsum.photos/256/?random=0
Request Method: GET
Status Code: 301 
```
![Screen Shot 2019-04-29 at 2 39 50 PM](https://user-images.githubusercontent.com/29871623/56921975-be2b8680-6a8c-11e9-9368-c009336b8510.png)

I can make the required changes and submit a PR 🎶

**Comment by nukeop** [comment]
Looks like they changed something, if you know how to fix it a pull request will be more than welcome.

**Comment by nukeop** [comment]
I think their CDN is down and it'll fix itself in a few hours.

**Comment by nukeop** [comment]
Fixed by #323, thanks for the help.
